### ISSUE TITLE
"reapply!" a whole style sheet without "always"

### ISSUE THREAD

**Issue by holgersindbaek** [motion-kit/motion-kit#132]
I've found myself in a situation where I've basically wrapped all my style code in "always" blocks, so I can update the view. 

Is it somehow possible to restyle a whole layout, without wrapping all your layout styles in "always" blocks?

**Comment by colinta** [comment]
No, not automatically.  You'd have to fork motion-kit to get that behavior.

**Comment by holgersindbaek** [comment]
Where would you start if you wanted to restyle the full layout?

-- 

All the best

Holger Sindbaek

 · UI/UX

 Designer & Web/iOS Programmer 

·

 

http://holgersindbaek.com/

On Wed, May 04, 2016 at 2:40 PM "Colin T.A. Gray"

<
mailto:
> wrote:

No, not automatically. You'd have to fork motion-kit to get that behavior.
